trio/_tests/test_path.py

```diff
@@ -114,7 +114,7 @@ async def test_compare_async_stat_methods(method_name):
 
 async def test_invalid_name_not_wrapped(path):
     with pytest.raises(AttributeError):
-        getattr(path, "invalid_fake_attr")
+        path.invalid_fake_attr  # noqa: B018
```

Comment:
hmmm. Does this not raise a type error that would also need to be silenced?
(Also probably good with a comment that B018 is `useless expression`, as mentioned in other comments)

Reply:
No, it should just be an AttributeError from `__getattr__()`.

Reply:
Changed to use `getattr` again in 2bd476d